languages/es_es/Settings/Vtiger.php

```diff
@@ -353,6 +353,19 @@
 	'LBL_COMPANY_LOGO_IN_SYSTEM' => 'Logotipo de la empresa visible en el sistema',
 	'LBL_COMPANY_LOGO_IN_LOGIN' => 'Logotipo de la empresa visible en el panel de inicio de sesión',
 	'LBL_HEIGHT_LOGO' => 'Altura del logotipo',
+	//Pluralized
+	'PLU_USERS_0' => 'Usuario',
+	'PLU_USERS_1' => 'Usuarios',
+	'PLU_USERS_2' => 'Usuarios',
```

Comment:
You don't have to add the translation for "_2" because your language doesn't have this plural form. 
Check this:
https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/vendor/yetiforce/Language.php#L187
http://docs.translatehouse.org/projects/localization-guide/en/latest/l10n/pluralforms.html?id=l10n/pluralforms#pluralforms-list